### ISSUE TITLE
Can't update date, "Unexpected token o in JSON at position 1"

### ISSUE THREAD

**Issue by MarlQ** [DasSauerkraut/calendar-weather#167]
Current foundry version: 0.7.5
I can not edit the date in any way, trying to save throws the following error:
![2020_10_31_17_30_57_FoundryVTT_Foundry_Virtual_Tabletop_•_A_Standalone_Virtual_Ta](https://user-images.githubusercontent.com/15791893/97784437-e70b6b00-1b9e-11eb-896f-9b1bdaffb20a.png)

I actually fixed this issue by changing in init.mjs
  ```
Hooks.on('calendarSettingsClose', (updatedData) => {
    console.log("calendar-weather | Closing Calendar form.");
    console.log(updatedData)
    c.rebuild(JSON.parse(updatedData));
    cwdtData.dt.genDateWordy();
    c.updateDisplay();
    c.updateSettings();
    c.settingsOpen(false);
  });
```

to 

```
  Hooks.on('calendarSettingsClose', async (updatedData) => {
    console.log("calendar-weather | Closing Calendar form.");
    updatedData = await updatedData;
    console.log(updatedData)
    c.rebuild(JSON.parse(updatedData));
    cwdtData.dt.genDateWordy();
    c.updateDisplay();
    c.updateSettings();
    c.settingsOpen(false);
  });
```

**Comment by MarlQ** [comment]
Yet I noticed, it _still_ doesn't update the time....

**Comment by geoidesic** [comment]
Fixed in this PR #163
